hnn_core/cell.py

```diff
@@ -269,10 +269,10 @@ class Cell:
         The synapses that the cell can use for connections.
     dipole_pp : list of h.Dipole()
         The Dipole objects (see dipole.mod).
-    rec_v : h.Vector()
+    rec_vsec : dict
         Recording of somatic voltage. Must be enabled
         by running simulate_dipole(net, record_vsoma=True)
-    rec_i : dict
+    rec_isec : dict
         Contains recording of somatic currents indexed
         by synapse type. (keys are soma_gabaa, soma_gabab etc.)
         Must be enabled by running simulate_dipole(net, record_isoma=True)
```

Comment:
Love this. Shouldn't the arguments in `simulated_dipole()` be `record_vsec=True` and `record_isec=True`, however?

Reply:
Totally right! Will push a fix